docs/.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.2
```

Comment:
What's this for? Looks like it could be a mistake.

Reply:
aha